uffizzi/.env.example

```diff
@@ -0,0 +1,40 @@
+APP_ENV=production
+APP_KEY=base64:kgk/4DW1vEVy7aEvet5FPp5un6PIGe/so8H0mvoUtW0=
+APP_DEBUG=true
+APP_LOG_LEVEL=debug
+APP_URL=http://crater.test
```

Comment:
Does this work on Uffizzi?

If this needs to be accurate, we could probably use the predictable URL's during the Render Compose step.

Reply:
Good point  :+1:  
Based on the original source of [.env.example](https://github.com/crater-invoice/crater/blob/master/.env.example) — it also have `APP_URL=http://crater.test`. 

I did not notice any mention/operation that replaces this environment variable in the original [docker-compose.yml](https://github.com/crater-invoice/crater/blob/master/docker-compose.yml). :thinking: 
That's why I didn't touch this `APP_URL` environment variable.